readthedocs/core/models.py

```diff
@@ -20,33 +16,34 @@ class UserProfile(TimeStampedModel):
 
     """Additional information about a User."""
 
-    # TODO: Overridden from TimeStampedModel just to allow null values,
-    # remove after deploy.
-    created = CreationDateTimeField(
-        _('created'),
-        null=True,
-        blank=True,
-    )
-    modified = ModificationDateTimeField(
```

Comment:
we need to create a data migration to migrate any null values, we could also just do this from the admin before running the migration

Reply:
I created a data migration for this.